doc/sslang-lrm.tex

```diff
@@ -715,7 +716,26 @@ \subsection{Sequences}
 
 
 \subsection{Pattern Matching}
-\fixme{todo}
+\begin{grammar}
+<exprBlk> ::= "match" <exprBlk> "{" <matchArms> "}"
+
+<matchArms> ::= matchArm "|" matchArms
+\alt matchArm
+
+<matchArm> ::= <pat> "=>" "{" <expr> "}"
+\end{grammar}
+
+Pattern matches are a succinct way to handle of a variety of cases for the expression in the match clause, known as the scrutinee. If the scrutinee satisfies the pattern in a match arm, the expression in the match arm is evaluated; otherwise, the expression in that arm is not evaluated.
+
+\begin{lstlisting}
+y = 0
+
+match x // Match clause
+    (Some a)  => y <- 1 // Arm if x is Some a
+    None      => y <- 2 // Arm if x is None
+\end{lstlisting}
+
+This expression checks if the scrutinee \texttt{x} is \texttt{Some a}; if it is, \texttt{y} is set to \texttt{1}. Otherwise, if \texttt{x} is \texttt{None},\texttt{y} is set to \texttt{2}. Pattern matches can also be expressed as \texttt{if-else} statements. However, pattern matches are a much more elegant way to handle multiple cases, especially when we are matching on types.
```

Comment:
I'd move this paragraph before the code listing and start off with "The below code sample checks...."